.travis.yml

```diff
@@ -22,6 +22,10 @@ matrix:
       env: TOXENV=py37
       dist: xenial
       sudo: true
+    - python: 3.7
+      env: TOXENV=docs
+      dist: xenial
+      sudo: true
```

Comment:
I'm not sure dist:xenial and sudo:true are needed these days to get Python 3.7 - could you please try without them?